website/docs/components/side-nav/partials/guidelines/guidelines.md

```diff
@@ -24,7 +24,7 @@ For cloud products, the icon set should always be the HCP service icon. For stan
 
 The body consists of a group of sections with vertical lists of links or buttons.
 
-### Section
+### List
```

Comment:
@cveigt I updated the naming and fixed the hierarchy of the List items header below.